test/e2e/e2e_test.go

```diff
@@ -30,12 +32,13 @@ const (
 	exGwAnnotation       = "k8s.ovn.org/hybrid-overlay-external-gw"
 )
 
-func checkContinuousConnectivity(f *framework.Framework, nodeName, podName, host string, port, timeout int, podChan chan *v1.Pod, errChan chan error) {
+func checkContinuousConnectivity(f *framework.Framework, nodeName, podName, host string, port int, podChan chan *v1.Pod, errChan chan error) {
+	defer GinkgoRecover()
 	contName := fmt.Sprintf("%s-container", podName)
 
 	command := []string{
 		"bash", "-c",
-		"set -xe; for i in {1..10}; do nc -vz -w " + strconv.Itoa(timeout) + " " + host + " " + strconv.Itoa(port) + "; sleep 2; done",
+		"set -xe; for i in {1..10}; do nc -vz " + host + " " + strconv.Itoa(port) + "; sleep 2; done",
```

Comment:
it seems that the test consist in spawning  this in background ,  deleting the ovn-kube pod meanwhile this is  running, and check that it didn't affect the connectivity.
Does agnhost connect run periodically or is a one shot connection?

Reply:
No, `agnhost connect` is a oneshot, like `nc`.

I was commenting on the fact that you removed the timeout from the `nc` args for no obvious reason. And then saying that if the reason that you did it was related to switching from busybox to agnhost, then you could bring the timeout back by using `agnhost connect` instead of `nc`.

Reply:
ahh, sorry, didn't get it the timeout was causing netcat to fail sporadically with some weird errors, so I removed to test it and I've found that my netcat had a default timeout, so no need to set it 
But now that you mention I realized that I'm using nmap.org/ncat that is different than the traditional netcat, I have to recover the timeout